app/views/companies/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="body-container">
+<h1 class="company-headline _container"><%= t('admins.companies.views.title') %></h1>
+<div class="wrapper-companies _container">
```

Comment:
minor comment: because on the first look seems like at the end of the file is a redundant '/div' tag
```suggestion
<div class="body-container">
  <h1 class="company-headline _container"><%= t('admins.companies.views.title') %></h1>
  <div class="wrapper-companies _container">
```

Reply:
fixed